index.js

```diff
@@ -66,6 +66,14 @@ function csurf (options) {
   var ignoreMethod = getIgnoredMethods(ignoreMethods)
 
   return function csrf (req, res, next) {
+    if (req.isInitialized) {
+      console.warn('[Warning] csurf() is duplicately called with same middleware in the cooke mode, first validation will result in the invalid token.')
```

Comment:
If this condition just won't work, what if we throe an error instead? That would make it much more obvious that there is an issue and since it won't work anyway, no need to actually continue.

Reply:
Hi, I've pushed new code to throw error.